recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set name = "dagster_dask" %}
-{% set version = "0.6.5" %}
+{% set name = "dagster-dask" %}
```

Comment:
Ah, that's why the bot hasn't picked it up. I'm not sure, probably we should create a new feedstock for it then.